packages/main/src/plugin/routes.ts

```diff
@@ -0,0 +1,21 @@
+export enum Page {
```

Comment:
I think we need to pickup a common pattern

filename is called routes
but there is a NavigateRequest object inside and a Page object

so it might be hard to know from these types come from

so might be interesting to use filename matching one of the object

like navigation.ts with NavigationRequest and NavigationPage objects (or better idea)